tests/eos-test-utils.c

```diff
@@ -775,6 +775,18 @@ generate_delta_files (GFile *repo,
   return cmd_result_ensure_ok (&cmd, error);
 }
 
+/**
+ * get_last_ref:
+ * @ref_to_commit: a #GHashTable mapping refs to commit numbers.
```

Comment:
Nitpick: Even though `g-ir-scanner` will never see this static method, I think it’s good to get into the habit of just adding GObject introspection annotations in all doc comments. So this should have `(element-type utf8 utf8)` (check that).